js/packages/openinference-core/test/trace/contextAttributes.test.ts

```diff
@@ -2,9 +2,11 @@ import { context, ContextManager } from "@opentelemetry/api";
 import { AsyncHooksContextManager } from "@opentelemetry/context-async-hooks";
 
 import {
+  METADATA,
```

Comment:
going to pr main to add support for these two tings then rebase / remove from feature branch